redbot/core/utils/menus.py

```diff
@@ -5,17 +5,108 @@
 import asyncio
 import contextlib
 import functools
+import logging
 import warnings
-from typing import Iterable, List, Optional, Union
+from typing import Iterable, List, Optional, Union, Dict
 import discord
+from ._dpy_menus_utils import dpymenu as _dpymenu
 
 from .. import commands
 from .predicates import ReactionPredicate
+from ..i18n import Translator
 
 _ReactableEmoji = Union[str, discord.Emoji]
 
+_ = Translator("Menus", __file__)
+
+log = logging.getLogger("red.menus")
+
 
 async def menu(
```

Comment:
This breaks existing users of this menu, it would be better to leave the existing menu as-is and just tell people to wait for stable or import it from vendor with a warning that it's not stable from there.